matchzoo/datapack.py

```diff
@@ -13,17 +12,28 @@ class DataPack(object):
     Matchzoo :class:`DataPack` data structure, store dataframe and context.
 
     Example:
-        >>> content = {
-        ...     'qid1':'query 1',
-        ...     'qid2':'query 2',
-        ...     'did1':'document 1',
-        ...     'did2':'document 2'
-        ... }
+        >>> left_data = [
+        ...     ['qid1', 'query 1', 'feature 1'],
```

Comment:
it's strange, `left_data` have `feature` while `right_data` not

Reply:
This is a real case where we only add features on query or document~